pkgs/games/dwarf-fortress/default.nix

```diff
@@ -35,6 +37,22 @@ let
   dfHackWorksWithCurrentVersion = true;
   dfHackEnabled = dfHackWorksWithCurrentVersion && enableDFHack;
 
+  themesThatWorkWithCurrentVersion = [
+    "phoebus"
+  ];
+  themeEnabled = builtins.elem theme themesThatWorkWithCurrentVersion;
+  themeToUse = if themeEnabled then theme else null;
+  themePkg = if (themeToUse != null)
+    then
+      import (./. + ("/themes/" + themeToUse + ".nix")) {
```

Comment:
Regarding the theme directory and using `mv`, I think I simply don't understand what you're trying to suggest. Do you want the user to supply an absolute path to a theme that they downloaded and extracted? That simply won't work. This theme, at least doesn't work if copied directly as it's extracted.